podaac/subsetter/gpm_cleanup.py

```diff
@@ -0,0 +1,65 @@
+"""
```

Comment:
Question for all: should we have a separate folder or module that has all collection- (or project-) specific logic?  I can't immediately think of a way to generalize this, and if we need some one-off logic, perhaps it should be separated a bit more from the generic subsetter module files?